Content.Server/AI/AimShootLifeProcessor.cs

```diff
@@ -119,7 +120,7 @@ private IEntity FindBestTarget()
 
                 // build the ray
                 var dir = entity.GetComponent<ITransformComponent>().WorldPosition - myTransform.WorldPosition;
-                var ray = new Ray(myTransform.WorldPosition, dir.Normalized);
+                var ray = new Ray(myTransform.WorldPosition, dir.Normalized, (int)CollisionGroup.Mob);
```

Comment:
This needs to collide with walls, the whole point of the ray is to check LOS with the player.
```suggestion
                var ray = new Ray(myTransform.WorldPosition, dir.Normalized, (int)(CollisionGroup.Mob | CollisionGroup.Grid));
```

Reply:
Alright, thought this was a way to look for mobs.